Signal/test/util/OWSOrphanDataCleanerTest.m

```diff
@@ -29,13 +37,9 @@ - (void)setUp
     }];
 
```

Comment:
You've made a bunch of changes to this test - is it still failing?

If it's passing can you remove the `BROKEN_TESTS` macro above?

Reply:
It's still failing, but I got us a little closer in this PR.  We'll get there!